locale/lt/LC_MESSAGES/crm.po

```diff
@@ -783,8 +789,9 @@ msgid ""
 "and quantity must be entered manually."
 msgstr ""
 "Norime pastebėti, kad šiuo atveju pasiūlymo eilutėse produktai bus ne "
-"tiesiog įrašomi, bet pasirenkami iš jūsų produktų sąrašo, kartu perkeliant ir "
-"produktų susijusią informaciją. Tik vieneto kainą ir kiekį reikės nurodyti patiems."
+"tiesiog įrašomi, bet pasirenkami iš jūsų produktų sąrašo, kartu "
+"perkeliant ir produktų susijusią informaciją. Tik vieneto kainą ir kiekį "
```

Comment:
Where are these changes coming from?

Reply:
No idea.

Reply:
This is CRM_offers, but I do not see any updates (Visual studio does not mark any changes).